src/components/conference/NewConference/Modal/ButtonFooterRow.svelte

```diff
@@ -34,6 +34,28 @@
         submitAndHandleModal(JSON.stringify(submitData), toastContext);
     }
 
+    async function gotoAddMoreDetailsPage() {
+        const enteredData = {
+            name: $name,
+            url: $url,
+            startDate: formatDateYYYYMMDD($startDate),
```

Comment:
Her får du en `Cannot read properties of null` dersom start eller slutt dato ikke er satt når du klikker 'Legg til mer info'

Reply:
La inn en sjekk i format-funksjonen